src/kernels/kernelFinder.ts

```diff
@@ -13,6 +13,11 @@ import { IKernelFinder, KernelConnectionMetadata } from './types';
  */
 @injectable()
 export class KernelFinder implements IKernelFinder {
+    private readonly _onDidChangeRegistrations = new EventEmitter<{
```

Comment:
Helps us determine when a remote has been registered, required in kernel picker (instead of having to poll, we can listten to an event)